source/developers/search-elasticsearch.rst

```diff
@@ -190,6 +190,30 @@ The final step is to trigger a new search when the user selects one of the value
    doSearch(queryParam, categories);
   });
 
+-----------------
+Multi-index Query
+-----------------
+
+Crafter CMS supports querying more than one Elasticsearch index in a single query.
+
+To search your site and other indexes, simply send a search query with a comma separated list of indexes/aliases (ES pointer to an index).  It will then search your site and the other indexes and return a single ranked result set
```

Comment:
I would remove `and return a single ranked result set` because it is not automatic, the user needs to set some properties in the search request to make it work that way